### PR TITLE
Add new tokenizer 

### PR DESCRIPTION
**Pull request by hirose125**
I added a new tokenizer for group selfies(https://github.com/aspuru-guzik-group/group-selfies)to tokenization.py. Since the input format is the same as selfies, I am using the selfies tokenizer as it is.

### CODE REVIEW

File: terminator/tokenization.py [hunk 1]
```diff
@@ -377,9 +379,11 @@ def __init__(
             self.text_tokenizer = RegexTokenizer(
                 regex_pattern=BIGSMILES_TOKENIZER_PATTERN
             )
+        elif self.language == "GroupSELFIES":
+            return sequence
```
Comment (drugilsberg) [inline terminator/tokenization.py:383]:
```suggestion
            self.text_tokenizer = SelfiesTokenizer()
```

**Review by drugilsberg — CHANGES_REQUESTED** [review]
Looks good, only a minor change

File: terminator/tokenization.py (hunk 1 above)
Comment (hirose125) [reply terminator/tokenization.py:383]: Thank you for review and advice

**Comment by drugilsberg** [comment]
close in favour of #18

**Pull request closed without merging**